app/src/main/java/com/stxnext/intranet2/activity/TimeReportActivity.java

```diff
@@ -50,7 +43,7 @@
 public class TimeReportActivity extends AppCompatActivity {
 
     public static final String USER_ID_TAG = "userId";
-    private String userId= null;
+    private String userId = null;
```

Comment:
Try not to commit code formatting changes and functional changes in one commit. 
One cannot distinct what really changed if all of them are in one place.